make/config/kyverno/policy.yaml

```diff
@@ -74,7 +74,7 @@ spec:
     name: require-drop-all
     preconditions:
       all:
-      - key: '{{ request.operation }}'
+      - key: '{{ request.operation || ''BACKGROUND'' }}'
```

Comment:
These changes are a result of re-running `kustomize build . > policy.yaml` which pulled in updated versions of the Kyverno policy files.